templates/taxi/driver_list.html

```diff
@@ -7,7 +7,11 @@ <h1>Driver list</h1>
       {% for driver in driver_list%}
         <li>
           <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
+          {% if driver.id == user.id %}
+            (Me)
+          {% else %}
+            ({{ driver.first_name }} {{ driver.last_name }})
+          {% endif %}
```

Comment:
> In Driver's list view add (Me), if this is a current user:
Something said that we need REPLACE full name by me?

As I see said `add (Me)`